samples/net/lwm2m_client/src/lwm2m-client.c

```diff
@@ -454,6 +456,76 @@ static void rd_client_event(struct lwm2m_ctx *client,
 	}
 }
 
+void my_timer_handler(struct k_timer *dummy)
+{
+	static int counter = 0;
+	if(counter == 5)
+	{
+		k_timer_stop(dummy);
+	}
+	counter++;
+
+
+	uint32_t random_int = sys_rand32_get();
+	double temp = random_int % 100;
+	LOG_WRN("Setting new temperature: %d", random_int%100);
+	lwm2m_engine_set_float("/3303/0/5700", &temp);
+}
+
+K_TIMER_DEFINE(my_timer, my_timer_handler, NULL);
```

Comment:
I think we should use more meaningful names than `my_timer`, `my_timer_handler`.